src/pages/EditScholarship.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import firebase from 'firebase';
+import { Container, Typography, Alert, AlertTitle } from '@mui/material';
+import ScholarshipForm from '../components/ScholarshipForm';
+import SubmissionAlert from '../components/SubmissionAlert';
+import Scholarships from '../models/Scholarships';
+import { Helmet } from 'react-helmet';
+
+function EditScholarship() {
+  const { id } = useParams();
+  const [scholarship, setScholarship] = useState(undefined);
+  const [error, setError] = useState();
+  const loading = !error && (!scholarship || !scholarship.data);
```

Comment:
I'm curious where the `!scholarship.data` comes from?

Reply:
Was most likely a check when we were passing scholarship in state.